scapy/data.py

```diff
@@ -209,24 +209,21 @@ def _resolve_MAC(self, mac):
             return ":".join([self[oui][0]] + mac.split(":")[3:])
         return mac
 
-    def __repr__(self):
-        return "\n".join("<%s %s, %s>" % (i[0], i[1][0], i[1][1]) for i in self.__dict__.items())  # noqa: E501
-
 
 def load_manuf(filename):
+    """Load manuf file from Wireshark.
+    param:
+     - filename: the file to load the manuf file from"""
     manufdb = ManufDA(_name=filename)
     with open(filename, "rb") as fdesc:
         for line in fdesc:
             try:
                 line = line.strip()
                 if not line or line.startswith(b"#"):
                     continue
-                oui, shrt = line.split()[:2]
-                i = line.find(b"#")
-                if i < 0:
-                    lng = shrt
-                else:
-                    lng = line[i + 2:]
+                parts = line.split()
+                oui, shrt = parts[:2]
+                lng = (b" ".join(parts[2:])).strip()
```

Comment:
Instead of ``parts = line.split()`` and ``b" ".join(parts[2:])``, it would be possible to use ``parts = line.split(None, 2)`` to split the line in at most 3 parts, and then use:

    lng = parts[2].lstrip(b'#').strip() if len(parts) > 2 else shrt

In order to use ``shrt`` if the long name is not defined, and to strip `#` if an old revision of Wireshark is used.

Reply:
Thanks. The new implementation should be cross-version compatible.

I’ll add unit tests to ensure that